std/lua/_std/haxe/Rest.hx

```diff
@@ -0,0 +1,71 @@
+package haxe;
+import lua.Lua.select;
+import lua.TableTools.maxn;
+import lua.TableTools.pack;
+import lua.Table;
+
+@:coreType
+abstract Rest<T> {
+	public var length(get,never):Int;
+    inline function get_length() : Int {
+        return maxn(cast this);
+    }
+
+	@:arrayAccess inline function get(index:Int):T {
+        return select(index+1, this);
+    }
+
+	@:to public inline function toArray():Array<T> {
+		return [for(i in 0...length) get(i)];
+	}
+
+    public inline function toString() : String {
+        return this.toArray().toString();
+    }
```

Comment:
Inconsistent tabs vs spaces in this file (maybe others?)

Reply:
It's probably time to run haxe-formatter over everything again. :)

Reply:
I retabbed, sorry about that.